dali/operators/decoder/video/video_decoder_base.h

```diff
@@ -82,7 +68,11 @@ class DLL_PUBLIC VideoDecoder : public Operator<Backend> {
     return shape;
   }
 
-  void DecodeSample(SampleView<Backend> output, FramesDecoder_t<Backend> &frames_decoder) {
+  /**
+   * @brief Decode sample with idx `s` to `output` tensor.
+   */
+  void DecodeSample(SampleView<OutBackend> output, int64_t s) {
```

Comment:
It is probably only me that feels that `s` as an index sounds odd. How about plain `idx`?